daemon/daemon.go

```diff
@@ -216,7 +225,23 @@ func (d *Daemon) Run() error {
 
 // Shutdown stops daemon.
 func (d *Daemon) Shutdown() error {
-	return d.server.Stop()
+	var errMsg string
```

Comment:
I am wondering if the variable `errMsg` is redundant.
I think with the current implementation, we can just return the returned err, right?

Reply:
@allencloud i want to execute all `stop` function even occurred an error when calling one of the  `stop` function